Ciac 5406 (#37906)
demisto/content: Packs/Mattermost/ReleaseNotes/2_0_5.md

#### Integrations

##### Mattermost v2

> Update location for the *Enable Incident Mirroring* field. It can now be found in the *Collect* section, under advanced settings.

Reviewer: ```suggestion
Updated the **Enable Incident Mirroring** parameter by moving it to the *Collect* section.
```
